EmotiBit.cpp

```diff
@@ -130,18 +131,16 @@ uint8_t EmotiBit::setup(String firmwareVariant)
 	Serial.println(firmware_version);
 	Serial.print("firmware_variant: ");
 	Serial.println(firmware_variant);
+	
+	// Wait for possible factory test init prompt
 	while (!Serial.available() && millis() - now < 2000)
 	{
 	}
 	while (Serial.available())
 	{
 		char input;
 		input = Serial.read();
-		if (input == 'R')
```

Comment:
I moved it to consolidate serial inputs to one location.
This *shouldn't* affect the enterprise wifi, because the reset option is still above the sensor setup and WiFi setup. But, I did not test it.

Reply:
Thinking more about, there is no reason we have a `reset` option downstream in the code.

If I weigh the pros and cons of having promts together in code, vs possibly breaking enterprise, preserving enterprise makes more sense.

So, I will add it back to this place. Will add a comment on why its there.